src/js/apps/programs/sidebar/action-sidebar_app.js

```diff
@@ -19,8 +20,28 @@ export default App.extend({
     }));
 
     this.showFormSharing();
-
     this.listenTo(this.action, 'change:_form change:outreach', this.showFormSharing);
+
+    this.showUploadsEnabled();
+    this.listenTo(this.action, 'change:allowed_uploads', this.showUploadsEnabled);
+  },
+  showUploadsEnabled() {
+    if (!Radio.request('bootstrap', 'setting', 'upload_attachments')) return;
+
+    const uploadsEnabledView = new UploadsEnabledView({
+      uploadsEnabled: !!size(this.action.get('allowed_uploads')),
```

Comment:
`size()` protects against `allowed_uploads` being `undefined`. Do we need to protect against that? Or can we assume an action will always have that attribute (`allowed_uploads: []`).

Reply:
Otherwise we could just do `this.action.get('allowed_uploads').length`.

Reply:
I think this should be `isUploadsEnabled`

Reply:
Ah yep, updated to that.